cpmpy/solvers/pysat.py

```diff
@@ -142,14 +141,16 @@ def __init__(self, cpm_model=None, subsolver=None):
         elif subsolver.startswith('pysat:'):
             subsolver = subsolver[6:] # strip 'pysat:'
 
-        # try to import the pblib module once
-        try:
-            from pysat import card, pb
-            self._card = card
-            self._pb = pb
-        except NameError:
-            self._card = None
-            self._pb = None
+        # try to import pypblib once (TODO I'd like class initialization, but can't find proper docs for it )
+        if CPM_pysat._pb is None:
```

Comment:
is there a reason to do this here? Another logical place would be in the classmethod supported(), which would then set cls._card and cls._pb?

Reply:
It should be a class initialization function which I've read somewhere but I can't for the life of me find the proper documentation..

Reply:
check the first line of code of this function (129)...

Reply:
Oops. fixed now.

Reply:
Ah I see what you mean. Yes I can put it there.

Reply:
Done